schemas/telemetry/downgrade/downgrade.4.schema.json

```diff
@@ -5,7 +5,7 @@
     "bq_metadata_format": "telemetry",
     "bq_table": "downgrade_v4",
     "expiration_policy": {
-      "delete_after_days": 10000
+      "delete_after_days": 400
```

Comment:
Does this mean we only keep the last 400 days of data or does it mean we delete the table after 400 days?